### DISCUSSION TITLE
LTS getting no compatability updates - does this mean no Fabric Loader 0.16.X support for 1.20.1?

### DISCUSSION THREAD

**Discussion by IdrisQe** [Sinytra/Connector#1433] [General]
Just curious. Wanted to test if a mod worked with Connector in 1.20.1, but it requires Fabric Loader 0.16.3, and the game won't even launch with beta45 of Connector as a result. Was going to post a bug report but then saw the notice about LTS only getting critical bugfixes.

On one hand, it's a compatability bug. On the other hand, having an entire major version of the loader which is supporter on that Minecraft version being incompatible feels like it would be considered a critical bug. So just thought I'd ask for confirmation.

![image](https://github.com/user-attachments/assets/80b58047-45cd-41e6-b6c3-793a09e44f4d)

**Comment by Su5eD** [comment]
I'll update it
